tools/shared.py

```diff
@@ -2800,6 +2800,7 @@ def read_and_preprocess(filename):
   # and shell.html is bigger than that!
   # See https://thraxil.org/users/anders/posts/2008/03/13/Subprocess-Hanging-PIPE-is-your-enemy/
   (path, file) = os.path.split(filename)
+  if not path: path = None
```

Comment:
we've been making progress to get these files passing flake8 style checks. i think for that, the body of the if should be on a new line